cmd/init/init.toml

```diff
@@ -0,0 +1,49 @@
+# One CLI to format the code tree - https://github.com/numtide/treefmt
+
+# Do not exit with error if a configured formatter is missing (env $TREEFMT_ALLOW_MISSING_FORMATTER)
+# allow-missing-formatter = true
+
+## Runs treefmt in a CI mode, enabling --no-cache, --fail-on-change and adjusting some other settings best suited to a
+## CI use case (env $TREEFMT_CI)
+# ci = true
+
+# The file into which a cpu profile will be written (env $TREEFMT_CPU_PROFILE)
+# cpu-profile = ./cpu.pprof
+
+# Exclude files or directories matching the specified globs (env $TREEFMT_EXCLUDES)
+# excludes = ["*.md", "*.gif"]
```

Comment:
Weren't the excludes under a [global] prefix? It could be a back-compat issue.

Reply:
If `excludes` is not set, it falls back to `global.excludes`.

This was driven in part for simplicity, also in part because viper has trouble binding flags to subsections, or at least I couldn't see how to easily achieve it.